src/test/java/com/google/step/finscholar/firebase/FirebaseStorageManagerTest.java

```diff
@@ -15,9 +17,13 @@
 @RunWith(JUnit4.class)
 public class FirebaseStorageManagerTest {
   // Constants for FirebaseStorageManagerTest.
+  /** Logger that sends logs to the Cloud Project console. */
+  private static final Logger log = Logger.getLogger(FirebaseStorageManager.class.getName());
   public static final String TEST_COLLECTION_NAME = "testObjects";
   public static final String TEST_DOCUMENT_NAME = "testDocument";
   public static final String EXPECTED_DOCUMENT_RETRIEVABLE = "{\"one\":\"testObjects\",\"two\":\"testDocument\"}";
+  public static final int TEST_BATCH_SIZE_LIMIT = 10;
```

Comment:
This batch size is larger than the number of test documents you have. Try to think of all the cases you need to  test to make sure you're grabbing the expected documents. I also don't see a test with no sort param set.

- first batch of collection where collection size > batch size
- middle batch of collection where collection size > batch size
- last batch of collection where # documents returns in < batch size